lighthouse-core/gather/gatherers/styles.js

```diff
@@ -115,9 +115,7 @@ class Styles extends Gatherer {
       Promise.all(contentPromises).then(styleHeaders => {
         driver.off('CSS.styleSheetAdded', this._onStyleSheetAdded);
         driver.off('CSS.styleSheetRemoved', this._onStyleSheetRemoved);
-        return driver.sendCommand('CSS.disable')
```

Comment:
We were bound to run into this eventually. Need to come up with a general solution.

At the very least for this PR, can you comment this out and make a note of why they're commented?